pytorch_lightning/testing/no_val_end_module.py

```diff
@@ -80,14 +80,14 @@ def loss(self, labels, logits):
         nll = F.nll_loss(logits, labels)
         return nll
 
-    def training_step(self, data_batch, batch_i):
+    def training_step(self, batch, batch_i):
```

Comment:
maybe also `bacth_i` -> `batch_idx` of just `idx`

Reply:
i -> idx is a good improvement